src/components/NavBar/NavBar.vue

```diff
@@ -59,9 +59,60 @@
       </v-autocomplete>
       <v-btn
         icon
-        class="mt-1"
+        class="Notification mt-1"
+        @click="test"
       >
-        <v-icon>mdi-bell</v-icon>
+        <template>
+          <div>
+            <v-menu
+              transition="slide-y-transition"
+              class="NotificationContainer"
+              bottom
+              offset-y
+            >
+              <template #activator="{ on, attrs }">
+                <v-btn
+                  icon
+                  v-bind="attrs"
+                  v-on="on"
+                >
+                  <v-icon>mdi-bell</v-icon>
+                </v-btn>
+              </template>
+              <v-list
+                v-if="notifItems.length===0"
+                three-line
+                class="NotificationListEmpty"
+              >
+                <v-list-item class="text-center">
+                  <v-col class="grey--text darken-3--text font-weight-bold">
+                    YOU DO NOT HAVE ANY NOTIFICATION YET
+                  </v-col>
+                </v-list-item>
+              </v-list>
+              <v-list
+                v-else
+                three-line
+                class="NotificationList"
+              >
+                <v-subheader class="subheader font-weight-bold mt-1 black--text">
+                  Lastest Notification
```

Comment:
Add s to notification

Reply:
And also remove the "s" from Lastest --> Latest

Reply:
fixed
![image](https://user-images.githubusercontent.com/37958655/158076797-6af8c830-31b9-471a-bf3c-0744ac4b107b.png)